R/Lab 1.Rmd

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
Yes I believe it is a duplicate.  I can save the location of the .rmd files to the correct location.  My apologies.  Also, the image paths are currently set to a folder on my computer.  My thought was once it is pushed onto github.com I can go in and reference them there so they will always be available.  Does that sound alright?